lib/utils.js

```diff
@@ -206,7 +206,13 @@ function getBowerContent(from) {
  * @param {requestCallback} callback - The callback that handles the response.
  */
 function getNpmRc(callback) {
-    let npmRcFilePath = path.join(os.homedir(), ".npmrc");
+    let npmRcFilePath;
+
+    if (process.env.NODE_ENV !== "test") {
+        npmRcFilePath = path.join(os.homedir(), ".npmrc");
+    } else {
+        npmRcFilePath = path.join(__dirname, "..", "test/data/.npmrc");
+    }
```

Comment:
OK, since this Pull Request is one month old, I will make it pass and log your comment in an enhancement issue @franknarf8 .